### PR TITLE
More swagger documentation

### LINKED ISSUE #51: Revisão documentação swagger

### PR DESCRIPTION
**Pull request by CarlosFeldmann**
- Identifying USs in swagger operation description
- More comments on services

### CODE REVIEW

**Review by Victor-Planas — APPROVED** [review]
Nice!! :heart_eyes:

File: src/main/java/br/com/mercadolivre/bootcampw2/grupo11/socialmeli/config/exceptions/dtos/FieldValidationError.java [hunk 1]
```diff
@@ -11,10 +11,10 @@
 
     @Schema(description = "The key of the field that the validation error occurred", example = "userName")
     private String field;
-    @Schema(description = "Description of the error that occurred", example = "must not be empty")
+    @Schema(description = "Description of the error that occurred", example = "must have at least 3 characters")
     private String message;
     @JsonInclude(JsonInclude.Include.NON_NULL)
-    @Schema(description = "The value that was passed to this field", example = "invalidValue")
+    @Schema(description = "The value that was passed to this field", example = "dé")
```
Comment (Giovannapls) [inline src/main/java/br/com/mercadolivre/bootcampw2/grupo11/socialmeli/config/exceptions/dtos/FieldValidationError.java]: dé? 🧐

**Review by Giovannapls — APPROVED** [review]
✍️🗿 noted

File: src/main/java/br/com/mercadolivre/bootcampw2/grupo11/socialmeli/config/exceptions/dtos/FieldValidationError.java (hunk 1 above)
Comment (CarlosFeldmann) [reply src/main/java/br/com/mercadolivre/bootcampw2/grupo11/socialmeli/config/exceptions/dtos/FieldValidationError.java]: Nickname of a person, example just a example that the field must at least contains 3 chars

Comment (lmarizmeli) [reply src/main/java/br/com/mercadolivre/bootcampw2/grupo11/socialmeli/config/exceptions/dtos/FieldValidationError.java]: dé? 🧑‍🚀

File: src/main/java/br/com/mercadolivre/bootcampw2/grupo11/socialmeli/repositories/SellerRepository.java [hunk 2]
```diff
@@ -16,4 +20,9 @@
     @Query("select count(post) from PromotionalPost post where post.seller = :seller")
     long countPromotionalPost(@Param("seller") Seller seller);
 
+    @Query("SELECT sl FROM Seller sl " +
+            "JOIN Follow fd ON sl = fd.seller " +
```
Comment (lmarizmeli) [inline src/main/java/br/com/mercadolivre/bootcampw2/grupo11/socialmeli/repositories/SellerRepository.java]: follow date doesn't exist anymore -> "fd"

Comment (lmarizmeli) [reply src/main/java/br/com/mercadolivre/bootcampw2/grupo11/socialmeli/repositories/SellerRepository.java]: só sendo sendo chato

**Comment by CarlosFeldmann** [comment]
Feito chefia, removi o "Dé"

**Review by lmarizmeli — APPROVED** [review]

**Pull request merged**